pkg/kube/apisix/apis/config/v2beta3/register.go

```diff
@@ -42,6 +42,16 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 	scheme.AddKnownTypes(SchemeGroupVersion,
 		&ApisixRoute{},
 		&ApisixRouteList{},
+		&ApisixUpstream{},
+		&ApisixUpstreamList{},
+		&ApisixTls{},
+		&ApisixTlsList{},
+		&ApisixClusterConfig{},
+		&ApisixClusterConfigList{},
+		&ApisixConsumer{},
+		&ApisixConsumerList{},
+		&ApisixPluginConfig{},
+		&ApisixPluginConfigList{},
```

Comment:
Can you add test case to cover this?

Reply:
No, I will modify how this file is maintained.

Reply:
Got it. And need to fix CI 😄

Reply:
I haven't thought of how to fix the CI problem for the time being.  In fact, I have already submitted the generated file 😅